Multiple feature requests from SRA pilot

koenderks opened the pull request:
- Allow for one decimal point in audit risk model custom percentages
- Allow for negative values in the data editor in the workflow
- Show items with negative values in the misstated items table
- Add possible ordering based on book values to selected items table
- Make the data editor smaller so that it takes up less screen
- Add option to evaluate against the overall materiality in the workflow analyses

lottemensink commented:
For the selected items table: do we only want to sort by book value? Or should auditors be able to sort based on for example ID as well? I am not sure which options would be convenient in practice.

lottemensink commented:
For the overall materiality: Do we want to be able to provide some kind of evaluation table for the overall materiality as well? It looks a bit hidden in the text now. But again, I am not entirely sure what people would prefer in practice.

koenderks commented:
> It is not counted (0x) whereas it should be counted 1x I think. The evaluation table seems to be fine, it is "counted" in calculating the taint, ub, p-value, etc.

Don't we want to count this 0x in the statistical evaluation (the evaluation table might be wrong then)? We check them integrally right?

> Again, the evaluation table does not seem to be affected (that is, the evaluation table does not view everything as misstated, but the misstated items table does)

I fixed this, it now shows:
<img width="318" alt="image" src="https://github.com/user-attachments/assets/fa358acf-20cd-43a8-93d2-7fd04167c4f4" />

> Then, a general thing I noted (this is also in the module as we have it now) is that if you open one sampling workflow after another sampling workflow, you get this:

I know, unfortunately this is unavoidable unless there is a check in the backend of JASP if this column already exists and then automatically change the name in the created column (which I cannot do...)

> For the selected items table: do we only want to sort by book value? Or should auditors be able to sort based on for example ID as well? I am not sure which options would be convenient in practice.

ID can also be a character (i.e., "hoi", "doei"), so that makes it impossible. To not keep the interface cluttered, let's keep it like this and not offer too much options.

> For the overall materiality: Do we want to be able to provide some kind of evaluation table for the overall materiality as well? It looks a bit hidden in the text now. But again, I am not entirely sure what people would prefer in practice.

I added it to the evaluation table now:
<img width="276" alt="image" src="https://github.com/user-attachments/assets/264672a8-1d6c-4974-9e64-06ce8d21e7c0" />

shun2wang commented:
i18n check now passed. thanks!
